pkg/database/connection.go

```diff
@@ -103,6 +104,22 @@ func (dao *DAO) InitializeTables() {
 		"CREATE INDEX IF NOT EXISTS data_name_idx ON search.resources USING GIN ((data ->  'name'))")
 	checkError(err, "Error creating index on search.resources data key name.")
 
+	createViewScript := strings.TrimSpace(`CREATE or REPLACE VIEW search.all_edges AS 
+	SELECT * from search.edges 
+	UNION
+	SELECT a.uid as sourceid , a.data->>'kind' as sourcekind, b.uid as destid, b.data->>'kind' as destkind, 
+	'deployedBy' as edgetype, a.cluster as cluster  
+	FROM search.resources a
+	INNER JOIN search.resources b
+	ON split_part(a.data->>'_hostingSubscription', '/', 1) = b.data->>'namespace'
+	AND split_part(a.data->>'_hostingSubscription', '/', 2) = b.data->>'name'
```

Comment:
Can hostingSubscription's format ever change? Or is there ever a case where there is no namespace provided?

Reply:
We don't expect it to change. This is the standard shared by app team.

Reply:
We have the table definitions part of search-v2-operator . I think we should have this also part of that
@SherinV @jlpadilla https://github.com/stolostron/search-v2-operator/blob/3cf610feb59ae21752b522b17de01c8524a2fda9/controllers/create_pgconfigmap.go#L26

Reply:
Well, we need to revisit where we will put the table definitions. The init script won't work when we allow to bring your own postgres.

For now keeping in both places reduces our chances of problems but it doesn't cover all scenarios.